docs/getting-started/README.md

```diff
@@ -0,0 +1,120 @@
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/sovity/edc-ui">
+    <img src="https://raw.githubusercontent.com/sovity/edc-ui/main/src/assets/images/sovity_logo.svg" alt="Logo" width="300">
+  </a>
+
+<h3 align="center">EDC-Connector:<br />Sovity EDC CE</h3>
+
+  <p align="center">
+    <a href="https://github.com/sovity/edc-extensions/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/sovity/edc-extensions/issues">Request Feature</a>
+  </p>
+</div>
+
+## Getting Started
+
+The following example shows you how to fully configure an MDS EDC.
+
+1. Ensure the [EDC Image](../../connector) you are using in the [docker-compose.yaml](../../docker-compose.yaml) file
+   is `edc-ce-mds`.
+2. Adjust the Connector Metadata:
+   ```yaml
+   # Connector Technical Name
+   MY_EDC_NAME_KEBAB_CASE: "example-connector"
+
+   # Connector Localized Name / Title
+   MY_EDC_TITLE: "EDC Connector"
+
+   # Connector Description Text
+   MY_EDC_DESCRIPTION: "Community Edition EDC Connector by sovity"
+
+   # Connector Curator
+   # The company using the EDC Connector, configuring data offers, etc.
+   MY_EDC_CURATOR_URL: "https://example.com"
+   MY_EDC_CURATOR_NAME: "Example GmbH"
+
+   # Connector Maintainer
+   # The company hosting the EDC Connector
+   MY_EDC_MAINTAINER_URL: "https://sovity.de"
+   MY_EDC_MAINTAINER_NAME: "sovity GmbH"
+   ```
+3. Generate your `.jks` file and your SKI/AKI ClientID with [generate_ski_aki.sh](secrets/generate_ski_aki.sh).
+4. Adjust your data space specific configuration:
+    ```yaml
+    # Required by the broker-extensions,
+    EDC_OAUTH_CLIENT_ID: '_your SKI/AKI_'
+    EDC_KEYSTORE: '_your keystore file relative to docs/getting-started/secrets_'
+    EDC_KEYSTORE_PASSWORD: '_your keystore password_'
+   
+    # MDS Test Environment
+    EDC_OAUTH_TOKEN_URL: 'https://daps.test.mobility-dataspace.eu/token'
+    EDC_OAUTH_PROVIDER_JWKS_URL: 'https://daps.test.mobility-dataspace.eu/jwks.json'
+    EDC_BROKER_BASE_URL: 'https://broker.test.mobility-dataspace.eu'
+    EDC_CLEARINGHOUSE_LOG_URL: 'https://clearing.test.mobility-dataspace.eu/messages/log'
+    ```
+5. Start the EDC
+   ```shell
+   # Log-in into the Github Container Registry
+   docker login ghcr.io
+   
+   # Fetches latest images
+   docker compose pull
+   
+   # Runs EDC and EDC UI at localhost:11000
+   docker compose up
+   ```
+6. Visit [localhost:11000](http://localhost:11000)
+
+## OpenAPI Yaml
+
+There's an [openapi.yaml](../openapi.yaml) that includes the vanilla EDC endpoints exposed by the current control- and
+data plane APIs.
+
+## Postman Collection
+
+To further try out our EDC Backend without the UI, try our Postman Collection:
+
+1. Use Postman (https://github.com/postmanlabs)
+2. Import the collection [docs/postman_collection.json](../postman_collection.json).
```

Comment:
Don't tell anyone, but the name of the Postman collection has changed, otherwise it will probably become a 404... :P